src/python/IndexChart.py

```diff
@@ -16,7 +16,7 @@
 try:
   from pygnuplot import gnuplot as Gnuplot
 except ImportError:
-  print('WARNING: Gnuplot module not present; will not make charts')
```

Comment:
Whoa, this is SUPER cleanup ;)  But this conflicts ... @rmuir already nuked this craziness.

Reply:
hm okay maybe we need a rebase here, but I never know how to deal with that in GH PRs; I feel like if I rebase it will erase all history

Reply:
Oh wait, I `git merge origin/main` and no conflicts? Maybe Robert's change has not landed yet?

Reply:
> hm okay maybe we need a rebase here, but I never know how to deal with that in GH PRs; I feel like if I rebase it will erase all history

Me neither!  I think rebase by definition means destroying / rewriting history since it must move your prior commits on top of the new ones?  Hmm then what does that do to the history on this issue (reviews w/ comments on specific lines of the to-be-rewritten commits...)?